src/check.py

```diff
@@ -135,9 +135,13 @@ def _update_action_problem(action: UpdateAction) -> Problem | None:
         )
 
     logging.error(
-        "there is a problem preventing the execution of an action, action: %s, problem: %s",
+        (
+            "there is a problem preventing the execution of an action\npath: %s\nproblem: %s\n"
+            "action: %s"
```

Comment:
Can we break the long string at the point of newlines? For example
```python
   (
        "there is a problem preventing the execution of an action\n"
        "path: %s\n"
        "problem: %s\n"
        "action: %s"
   )

Reply:
Good point, fixed